stripe/src/main/java/com/stripe/android/StripeApiHandler.java

```diff
@@ -729,6 +804,16 @@ static String getRetrieveTokenApiUrl(@NonNull String tokenId) {
         return String.format(Locale.ROOT, "%s/%s", getApiUrl(), tokenId);
     }
 
+    @VisibleForTesting
+    static String getRetrieveIssuingCardPinUrl(@NonNull String cardId) {
+        return String.format(Locale.ENGLISH, "%s/v1/issuing/cards/%s/pin", LIVE_API_BASE, cardId);
+    }
+
+    @VisibleForTesting
+    static String getUpdateIssuingCardPinUrl(@NonNull String cardId) {
+        return String.format(Locale.ENGLISH, "%s/v1/issuing/cards/%s/pin", LIVE_API_BASE, cardId);
```

Comment:
use `Locale.ROOT` to be consistent with other places where we format strings

Reply:
👍